tools/mason/mason.chpl

```diff
@@ -0,0 +1,152 @@
+
+use FileSystem;
+use Help;
+use Spawn;
+use TOML;
```

Comment:
Be sure to update the `use` statements here, once commands are distributed into other modules. e.g. Do we still need `Spawn` and `TOML` here?